packages/core-bridge/src/conversions.rs

```diff
@@ -176,38 +177,25 @@ impl ObjectHandleConversionsExt for Handle<'_, JsObject> {
 
     fn as_telemetry_options(&self, cx: &mut FunctionContext) -> NeonResult<TelemetryOptions> {
         let mut telemetry_opts = TelemetryOptionsBuilder::default();
-
-        if let Some(tf) = js_optional_value_getter!(cx, self, "tracingFilter", JsString) {
-            telemetry_opts.tracing_filter(tf);
-        }
         telemetry_opts.no_temporal_prefix_for_metrics(
             js_optional_value_getter!(cx, self, "noTemporalPrefixForMetrics", JsBoolean)
                 .unwrap_or_default(),
         );
         if let Some(ref logging) = js_optional_getter!(cx, self, "logging", JsObject) {
-            if let Some(_) = get_optional(cx, logging, "console") {
-                telemetry_opts.logging(Logger::Console);
-            } else if let Some(ref forward) = js_optional_getter!(cx, logging, "forward", JsObject)
-            {
-                let level = js_value_getter!(cx, forward, "level", JsString);
-                match LevelFilter::from_str(&level) {
-                    Ok(level) => {
-                        telemetry_opts.logging(Logger::Forward(level));
-                    }
-                    Err(err) => cx.throw_type_error(format!(
-                        "Invalid telemetryOptions.logging.forward.level: {}",
-                        err
-                    ))?,
-                }
+            let filter = js_value_getter!(cx, logging, "filter", JsString);
+            if get_optional(cx, logging, "console").is_some() {
+                telemetry_opts.logging(Logger::Console { filter });
+            } else if get_optional(cx, logging, "forward").is_some() {
+                telemetry_opts.logging(Logger::Forward { filter });
```

Comment:
We don't support the deprecated `level` option anymore, please fix.

Reply:
Backward compatibility and default values are handled in [runtime.ts](https://github.com/temporalio/sdk-typescript/pull/963/files#diff-b144ddafebfd1e9ede8c1761ec8b821fd2611f017a5f0d88d1faac7362752e3eR216-R217).

If present, the `level` property is converted to a filter string. The rust code only sees see compiled options, matching the new format.

Reply:
Note that since we now have a single filter for logging, if both `forward.level` and `tracingFilter` were previously defined, then the `tracingFilter` will be silently ignored. AFAIK, we can't do any better there.